### PR TITLE
[Wiki] Incorporated feedback of contributor

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Description

- Incorporated feedback of Contributor (especially for onboarding scenario 2)

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Update to documentation

### CODE REVIEW

File: docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md [hunk 1]
```diff
@@ -73,13 +77,17 @@ Alternatively, you can also do the same with a specific release by navigating to
 
   <img src="./media/SetupEnvironment/downloadZipRelease.png" alt="Download zip" height="150">
 
+To update your copy with the latest changes in CARML, you would need to download, for example, the latest CARML release the same way you did during your initial setup, copy its folder content to your repository's location, and upload the changes.
```
Comment (eriqua) [inline docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md:80]: Shouldn't this go in the "Fetch latest changes" section?

File: docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md [hunk 2]
```diff
@@ -305,7 +312,7 @@ The primary pipeline settings file ([`settings.yml`](https://github.com/Azure/Re
 
 | Variable Name | Example Value | Description |
 | - | - | - |
-| `templateSpecsRGName` | `'artifacts-rg'` | The resource group to host the created template-specs. |
+| `templateSpecsRGName` | `'artifacts-rg'` | The resource group to host the created template-specs. </p> Will be automatically created of not yet existing in the target subscription. |
```
Comment (eriqua) [inline docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md:315]:
```suggestion
| `templateSpecsRGName` | `'artifacts-rg'` | The resource group to host the created template-specs. </p> Will be automatically created if not yet existing in the target subscription. |
```

File: docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md [hunk 3]
```diff
@@ -317,8 +324,8 @@ The primary pipeline settings file ([`settings.yml`](https://github.com/Azure/Re
 
 | Variable Name | Example Value | Description |
 | - | - | - |
-| `bicepRegistryName` | `'adpsxxazacrx001'` | The container registry to publish Bicep templates to. <p> **NOTE:** Must be globally unique. |
-| `bicepRegistryRGName` | `'artifacts-rg'` | The resource group of the container registry to publish Bicep templates to. Is used to create a new container registry if not yet existing. |
+| `bicepRegistryName` | `'adpsxxazacrx001'` | The container registry to publish Bicep templates to. </p> **NOTE:** Must be globally unique. </p> Will be automatically created of not yet existing in the target subscription. |
+| `bicepRegistryRGName` | `'artifacts-rg'` | The resource group of the container registry to publish Bicep templates to. Is used to create a new container registry if not yet existing. </p> Will be automatically created of not yet existing in the target subscription. |
```
Comment (eriqua) [inline docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md:328]:
```suggestion
| `bicepRegistryName` | `'adpsxxazacrx001'` | The container registry to publish Bicep templates to. </p> **NOTE:** Must be globally unique. </p> Will be automatically created if not yet existing in the target subscription. |
| `bicepRegistryRGName` | `'artifacts-rg'` | The resource group of the container registry to publish Bicep templates to. Is used to create a new container registry if not yet existing. </p> Will be automatically created if not yet existing in the target subscription. |
```

File: docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md [hunk 4]
```diff
@@ -361,9 +368,9 @@ This section will explain what is required to publish the modules to [Azure Arti
...
 
-#### Implementation Guidance
+<h4><b>Implementation Guidance</b></h4>
 
-Each `./azuredevops/modulePipelines` YAML pipeline already calls [`/.azuredevops/pipelineTemplates/jobs.publishModule.yml`](https://github.com/Azure/ResourceModules/blob/main/.azuredevops/pipelineTemplates/jobs.publishModule.yml). This YAML template contains a method to `Publish module to artifacts feed` via [`utilities\pipelines\resourcePublish\Publish-ModuleToUniversalArtifactsFeed.ps1`](https://github.com/Azure/ResourceModules/blob/main/utilities\pipelines\resourcePublish\Publish-ModuleToUniversalArtifactsFeed.ps1).
+Each `./azuredevops/modulePipelines` YAML pipeline invokes the [`/.azuredevops/pipelineTemplates/jobs.publishModule.yml`](https://github.com/Azure/ResourceModules/blob/main/.azuredevops/pipelineTemplates/jobs.publishModule.yml) template. This YAML template contains a method to `Publish module to artifacts feed` via the[`utilities\pipelines\resourcePublish\Publish-ModuleToUniversalArtifactsFeed.ps1`](https://github.com/Azure/ResourceModules/blob/main/utilities\pipelines\resourcePublish\Publish-ModuleToUniversalArtifactsFeed.ps1) script, which ultimately publishes the modules.
```
Comment (eriqua) [inline docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md:373]:
```suggestion
Each `./azuredevops/modulePipelines` YAML pipeline invokes the [`/.azuredevops/pipelineTemplates/jobs.publishModule.yml`](https://github.com/Azure/ResourceModules/blob/main/.azuredevops/pipelineTemplates/jobs.publishModule.yml) template. This YAML template contains a method to `Publish module to artifacts feed` via the [`utilities\pipelines\resourcePublish\Publish-ModuleToUniversalArtifactsFeed.ps1`](https://github.com/Azure/ResourceModules/blob/main/utilities\pipelines\resourcePublish\Publish-ModuleToUniversalArtifactsFeed.ps1) script, which ultimately publishes the modules.
```

File: docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md (hunk 1 above)
Comment (AlexanderSehr) [reply docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md:80]: Yes-ish. Apparently it wasn't obvious enough - so I would suggest to have this here with a link to the 'fetch changes' section for 'further information'.

Comment (AlexanderSehr) [reply docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md:80]: Update: Added it below.

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**